bin/janus

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Process\ExecutableFinder;
+use Symfony\Component\Process\Process;
+
+require_once $_composer_autoload_path ?? (__DIR__ . '/../vendor/autoload.php');
+
+$input = new ArgvInput();
+$output = new ConsoleOutput();
+
+$allowedCommands = [
+	"init-symfony" => null,
+];
+
+
+(new SingleCommandApplication())
+	->setName("Janus")
+	->addArgument(
+		"command",
+		InputArgument::REQUIRED,
+		suggestedValues: array_keys($allowedCommands),
+	)
+	->setCode(
+		function (InputInterface $input, OutputInterface $output) : int
+		{
+			$io = new SymfonyStyle($input, $output);
+			$io->title("Janus");
+			$command = $input->getArgument("command");
+
+			try
+			{
+				return match ($command)
+				{
+					"init-symfony" => initializeSymfony($io),
+					null => printUsage(),
+					default => printError($io, "Unknown command: {$command}"),
+				};
+			}
+			catch (Throwable $exception)
+			{
+				$io->error("Running janus failed: {$exception->getMessage()}");
+				return 2;
+			}
+		}
+	)
+	->run();
+
+return;
+
+
+// region Commands
+function initializeSymfony (SymfonyStyle $io) : int
+{
+	$io->writeln("• Copying config files to the project...");
+	copyFilesIntoProject($io, "symfony");
+
+	$io->writeln("• Updating <fg=yellow>composer.json</>...");
+	addToProjectComposerJson([
```

Comment:
How should we handle our Private Packagist configs? Do people still have to manually copy it into their projects?

Reply:
What does that have to do with the changes here?